src/analyze/gui/parser.cpp

```diff
@@ -647,16 +658,37 @@ void Parser::parseImpl(const QString& path, const QString& diffBase, const Filte
     auto oldData = std::move(m_data);
     using namespace ThreadWeaver;
     stream() << make_job([this, oldData, path, diffBase, filterParameters]() {
+        const auto isReparsing = (path == m_path && oldData && diffBase.isEmpty());
+        auto parsingMsg = isReparsing ? i18n("reparsing data") : i18n("parsing data");
+        emit progressMessageAvailable(parsingMsg);
+
+        auto updateProgress = [this, parsingMsg](const ParserData & data){
```

Comment:
remove space before `&` please

Reply:
afaik there's a `.clang-format` you could try to use?

Reply:
Done. (ran clang-format-diff to avoid formatting the whole file as there were unrelated items that ended up formatted.)